charts/osm/crds/meshconfig.yaml

```diff
@@ -187,6 +187,10 @@ spec:
                       description: Sets the service certificate validity duration, represented as a sequence of decimal numbers each with optional fraction and a unit suffix.
                       type: string
                       default: "24h"
+                clusterId:
```

Comment:
We want to avoid making changes to the meshconfig for multicluster work since this is an experimental feature. Would it be possible to add this else where ?

Reply:
How about creating a `experimental` section in MeshConfig?

Reply:
Sure, so nest this under a new `experimental` property?